src/components/AmbassadorCard.tsx

```diff
@@ -251,6 +255,22 @@ export default forwardRef(function AmbassadorCard(
             <p>{ambassador.native.text}</p>
           </div>
 
+          <div>
+            <h3 className={headingClass}>Lifespan</h3>
+            <p>
+              Wild:{" "}
+              {"wild" in ambassador.lifespan
+                ? `${stringifyLifespan(ambassador.lifespan.wild)} years`
+                : "Unknown"}
+            </p>
+            <p>
+              Captivity:{" "}
+              {"captivity" in ambassador.lifespan
+                ? `${stringifyLifespan(ambassador.lifespan.captivity)} years`
+                : "Unknown"}
+            </p>
```

Comment:
For these, could we use the same tilde with the approx title that we use for a few other bits here?